[2024-10-03 22:45:51,205: INFO: main: welcome to electorals and sysmbols detection project]
[2024-11-09 13:06:42,896: INFO: main: welcome to electorals and sysmbols detection project]
[2024-11-10 12:44:58,637: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-10 12:44:58,641: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-10 12:45:29,765: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-10 12:45:29,766: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-10 12:45:32,644: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-10 12:45:32,646: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-10 12:50:47,445: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-10 12:50:47,446: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-10 12:53:24,657: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-10 12:53:24,659: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-10 12:53:24,660: INFO: common: created directory at: artifacts]
[2024-11-10 12:53:24,665: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-10 12:53:24,666: INFO: 797920417: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-10 12:53:38,023: INFO: 797920417: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-10 13:00:59,899: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-10 13:00:59,901: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-10 13:00:59,902: INFO: common: created directory at: artifacts]
[2024-11-10 13:00:59,903: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-10 13:00:59,903: INFO: 1728023487: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-10 18:36:02,663: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-10 18:36:02,664: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-10 18:36:02,666: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-10 18:36:02,667: INFO: common: created directory at: artifacts]
[2024-11-10 18:36:02,667: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-10 18:36:02,669: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-10 18:36:16,728: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-10 18:36:19,727: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<

x==================x]
[2024-11-13 02:48:52,067: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-13 02:48:52,067: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-13 02:48:52,081: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-13 02:48:52,082: INFO: common: created directory at: artifacts]
[2024-11-13 02:48:52,082: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-13 02:48:52,082: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-13 02:49:05,256: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-13 02:49:06,159: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-16 14:39:39,095: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 14:39:39,111: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 14:39:39,113: INFO: common: created directory at: artifacts]
[2024-11-16 14:41:27,380: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 14:41:27,381: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 14:41:27,382: INFO: common: created directory at: artifacts]
[2024-11-16 15:33:02,745: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 15:33:02,747: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 15:33:02,748: INFO: common: created directory at: artifacts]
[2024-11-16 15:34:01,003: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 15:34:01,005: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 15:34:01,006: INFO: common: created directory at: artifacts]
[2024-11-16 15:34:03,799: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 15:34:03,801: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 15:34:03,802: INFO: common: created directory at: artifacts]
[2024-11-16 15:36:43,933: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 15:36:43,936: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 15:36:43,937: INFO: common: created directory at: artifacts]
[2024-11-16 15:41:40,861: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 15:41:40,866: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 15:41:40,867: INFO: common: created directory at: artifacts]
[2024-11-16 15:46:03,108: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 15:46:03,111: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 15:46:03,113: INFO: common: created directory at: artifacts]
[2024-11-16 15:59:34,211: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 15:59:34,224: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 15:59:34,225: INFO: common: created directory at: artifacts]
[2024-11-16 16:05:20,428: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:05:20,431: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:05:20,432: INFO: common: created directory at: artifacts]
[2024-11-16 16:10:47,124: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:10:47,127: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:10:47,129: INFO: common: created directory at: artifacts]
[2024-11-16 16:18:41,663: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:18:41,665: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:18:41,666: INFO: common: created directory at: artifacts]
[2024-11-16 16:19:00,949: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:19:00,949: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:19:00,949: INFO: common: created directory at: artifacts]
[2024-11-16 16:19:11,557: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:19:11,560: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:19:11,562: INFO: common: created directory at: artifacts]
[2024-11-16 16:20:41,422: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:20:41,426: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:20:41,428: INFO: common: created directory at: artifacts]
[2024-11-16 16:25:06,584: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:25:06,584: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:25:06,595: INFO: common: created directory at: artifacts]
[2024-11-16 16:25:18,025: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:25:18,029: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:25:18,030: INFO: common: created directory at: artifacts]
[2024-11-16 16:30:00,847: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:30:00,849: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:30:00,851: INFO: common: created directory at: artifacts]
[2024-11-16 16:31:03,996: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-16 16:31:04,006: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-16 16:31:04,007: INFO: common: created directory at: artifacts]
[2024-11-17 12:08:13,138: INFO: font_manager: generated new fontManager]
[2024-11-17 14:12:51,734: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 14:12:51,744: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 14:12:51,745: INFO: common: created directory at: artifacts]
[2024-11-17 15:39:24,116: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 15:39:24,118: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 15:39:24,122: INFO: common: created directory at: artifacts]
[2024-11-17 15:40:25,008: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 15:40:25,013: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 15:40:25,013: INFO: common: created directory at: artifacts]
[2024-11-17 15:40:34,747: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 15:40:34,749: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 15:40:34,749: INFO: common: created directory at: artifacts]
[2024-11-17 15:41:19,506: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 15:41:19,509: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 15:41:19,509: INFO: common: created directory at: artifacts]
[2024-11-17 17:04:48,233: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 17:04:48,237: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 17:04:48,239: INFO: common: created directory at: artifacts]
[2024-11-17 17:31:14,094: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 17:31:14,096: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 17:31:14,097: INFO: common: created directory at: artifacts]
[2024-11-17 19:25:59,139: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 19:25:59,147: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 19:25:59,149: INFO: common: created directory at: artifacts]
[2024-11-17 19:30:30,633: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 19:30:30,636: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 19:30:30,637: INFO: common: created directory at: artifacts]
[2024-11-17 21:01:41,105: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 21:01:41,107: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 21:01:41,109: INFO: common: created directory at: artifacts]
[2024-11-17 21:56:26,097: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 21:56:26,099: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 21:56:26,101: INFO: common: created directory at: artifacts]
[2024-11-17 22:40:38,570: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-17 22:40:38,573: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-17 22:40:38,575: INFO: common: created directory at: artifacts]
[2024-11-18 08:26:09,415: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 08:26:09,423: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 08:26:09,425: INFO: common: created directory at: artifacts]
[2024-11-18 14:53:22,127: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 14:53:22,140: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 14:53:22,140: INFO: common: created directory at: artifacts]
[2024-11-18 15:03:24,513: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:03:24,517: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:03:24,519: INFO: common: created directory at: artifacts]
[2024-11-18 15:03:41,327: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:03:41,330: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:03:41,334: INFO: common: created directory at: artifacts]
[2024-11-18 15:05:27,329: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:05:27,335: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:05:27,336: INFO: common: created directory at: artifacts]
[2024-11-18 15:06:55,812: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:06:55,816: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:06:55,817: INFO: common: created directory at: artifacts]
[2024-11-18 15:09:17,594: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:09:17,599: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:09:17,600: INFO: common: created directory at: artifacts]
[2024-11-18 15:09:51,565: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:09:51,570: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:09:51,571: INFO: common: created directory at: artifacts]
[2024-11-18 15:13:32,406: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:13:32,410: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:13:32,411: INFO: common: created directory at: artifacts]
[2024-11-18 15:13:56,281: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:13:56,285: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:13:56,288: INFO: common: created directory at: artifacts]
[2024-11-18 15:15:08,933: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:15:08,936: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:15:08,936: INFO: common: created directory at: artifacts]
[2024-11-18 15:15:41,386: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:15:41,386: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:15:41,390: INFO: common: created directory at: artifacts]
[2024-11-18 15:15:50,710: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:15:50,712: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:15:50,716: INFO: common: created directory at: artifacts]
[2024-11-18 15:15:56,889: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:15:56,893: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:15:56,896: INFO: common: created directory at: artifacts]
[2024-11-18 15:17:02,070: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:17:02,073: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:17:02,076: INFO: common: created directory at: artifacts]
[2024-11-18 15:17:34,869: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:17:34,872: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:17:34,872: INFO: common: created directory at: artifacts]
[2024-11-18 15:18:54,413: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:18:54,417: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:18:54,418: INFO: common: created directory at: artifacts]
[2024-11-18 15:19:24,212: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:19:24,216: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:19:24,218: INFO: common: created directory at: artifacts]
[2024-11-18 15:20:09,270: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:20:09,273: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:20:09,273: INFO: common: created directory at: artifacts]
[2024-11-18 15:20:58,026: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:20:58,030: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:20:58,031: INFO: common: created directory at: artifacts]
[2024-11-18 15:22:45,400: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:22:45,405: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:22:45,409: INFO: common: created directory at: artifacts]
[2024-11-18 15:23:07,571: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:23:07,575: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:23:07,577: INFO: common: created directory at: artifacts]
[2024-11-18 15:24:11,366: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:24:11,370: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:24:11,370: INFO: common: created directory at: artifacts]
[2024-11-18 15:24:21,261: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:24:21,262: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:24:21,266: INFO: common: created directory at: artifacts]
[2024-11-18 15:24:46,012: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:24:46,018: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:24:46,020: INFO: common: created directory at: artifacts]
[2024-11-18 15:25:58,679: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:25:58,682: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:25:58,685: INFO: common: created directory at: artifacts]
[2024-11-18 15:26:23,504: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:26:23,509: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:26:23,511: INFO: common: created directory at: artifacts]
[2024-11-18 15:26:54,577: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:26:54,584: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:26:54,585: INFO: common: created directory at: artifacts]
[2024-11-18 15:28:30,379: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:28:30,379: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:28:30,384: INFO: common: created directory at: artifacts]
[2024-11-18 15:28:43,837: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:28:43,843: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:28:43,844: INFO: common: created directory at: artifacts]
[2024-11-18 15:28:52,294: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:28:52,299: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:28:52,300: INFO: common: created directory at: artifacts]
[2024-11-18 15:29:02,638: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:29:02,641: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:29:02,643: INFO: common: created directory at: artifacts]
[2024-11-18 15:30:26,845: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:30:26,849: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:30:26,849: INFO: common: created directory at: artifacts]
[2024-11-18 15:38:49,652: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:38:49,655: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:38:49,656: INFO: common: created directory at: artifacts]
[2024-11-18 15:49:23,856: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 15:49:23,858: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 15:49:23,860: INFO: common: created directory at: artifacts]
[2024-11-18 16:48:22,234: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:48:22,237: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:48:22,239: INFO: common: created directory at: artifacts]
[2024-11-18 16:48:40,527: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:48:40,527: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:48:40,527: INFO: common: created directory at: artifacts]
[2024-11-18 16:49:31,422: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:49:31,425: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:49:31,426: INFO: common: created directory at: artifacts]
[2024-11-18 16:51:51,377: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:51:51,381: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:51:51,382: INFO: common: created directory at: artifacts]
[2024-11-18 16:52:03,960: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:52:03,960: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:52:03,960: INFO: common: created directory at: artifacts]
[2024-11-18 16:52:21,198: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:52:21,200: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:52:21,202: INFO: common: created directory at: artifacts]
[2024-11-18 16:52:49,074: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:52:49,077: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:52:49,079: INFO: common: created directory at: artifacts]
[2024-11-18 16:52:56,580: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:52:56,582: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:52:56,582: INFO: common: created directory at: artifacts]
[2024-11-18 16:53:06,820: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:53:06,822: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:53:06,823: INFO: common: created directory at: artifacts]
[2024-11-18 16:55:59,884: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:55:59,888: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:55:59,890: INFO: common: created directory at: artifacts]
[2024-11-18 16:56:11,702: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:56:11,702: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:56:11,702: INFO: common: created directory at: artifacts]
[2024-11-18 16:56:24,656: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:56:24,660: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:56:24,660: INFO: common: created directory at: artifacts]
[2024-11-18 16:56:53,795: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 16:56:53,797: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 16:56:53,798: INFO: common: created directory at: artifacts]
[2024-11-18 17:09:56,969: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:09:56,972: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:09:56,973: INFO: common: created directory at: artifacts]
[2024-11-18 17:10:45,091: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:10:45,093: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:10:45,095: INFO: common: created directory at: artifacts]
[2024-11-18 17:13:19,818: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:13:19,821: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:13:19,821: INFO: common: created directory at: artifacts]
[2024-11-18 17:15:47,763: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:15:47,765: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:15:47,766: INFO: common: created directory at: artifacts]
[2024-11-18 17:18:59,647: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:18:59,649: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:18:59,651: INFO: common: created directory at: artifacts]
[2024-11-18 17:19:42,927: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:19:42,929: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:19:42,931: INFO: common: created directory at: artifacts]
[2024-11-18 17:20:30,192: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:20:30,195: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:20:30,196: INFO: common: created directory at: artifacts]
[2024-11-18 17:21:00,366: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:21:00,369: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:21:00,370: INFO: common: created directory at: artifacts]
[2024-11-18 17:21:27,876: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:21:27,879: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:21:27,880: INFO: common: created directory at: artifacts]
[2024-11-18 17:21:35,114: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:21:35,116: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:21:35,118: INFO: common: created directory at: artifacts]
[2024-11-18 17:22:47,231: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:22:47,233: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:22:47,233: INFO: common: created directory at: artifacts]
[2024-11-18 17:22:56,586: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:22:56,589: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:22:56,589: INFO: common: created directory at: artifacts]
[2024-11-18 17:25:28,927: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:25:28,930: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:25:28,930: INFO: common: created directory at: artifacts]
[2024-11-18 17:30:35,397: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:30:35,397: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:30:35,404: INFO: common: created directory at: artifacts]
[2024-11-18 17:31:05,481: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:31:05,483: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:31:05,485: INFO: common: created directory at: artifacts]
[2024-11-18 17:33:48,139: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:33:48,144: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:33:48,147: INFO: common: created directory at: artifacts]
[2024-11-18 17:34:24,152: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:34:24,155: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:34:24,156: INFO: common: created directory at: artifacts]
[2024-11-18 17:35:16,933: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:35:16,936: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:35:16,937: INFO: common: created directory at: artifacts]
[2024-11-18 17:35:38,848: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:35:38,853: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:35:38,856: INFO: common: created directory at: artifacts]
[2024-11-18 17:38:39,074: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:38:39,078: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:38:39,080: INFO: common: created directory at: artifacts]
[2024-11-18 17:38:55,432: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:38:55,432: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:38:55,432: INFO: common: created directory at: artifacts]
[2024-11-18 17:56:27,523: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:56:27,526: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:56:27,527: INFO: common: created directory at: artifacts]
[2024-11-18 17:58:38,392: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:58:38,396: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:58:38,396: INFO: common: created directory at: artifacts]
[2024-11-18 17:58:51,673: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:58:51,676: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:58:51,676: INFO: common: created directory at: artifacts]
[2024-11-18 17:59:11,890: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:59:11,890: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:59:11,890: INFO: common: created directory at: artifacts]
[2024-11-18 17:59:55,679: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 17:59:55,683: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 17:59:55,685: INFO: common: created directory at: artifacts]
[2024-11-18 18:11:07,553: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 18:11:07,556: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 18:11:07,557: INFO: common: created directory at: artifacts]
[2024-11-18 18:11:32,573: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 18:11:32,576: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 18:11:32,577: INFO: common: created directory at: artifacts]
[2024-11-18 18:12:30,429: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 18:12:30,432: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 18:12:30,433: INFO: common: created directory at: artifacts]
[2024-11-18 18:16:11,311: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 18:16:11,313: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 18:16:11,314: INFO: common: created directory at: artifacts]
[2024-11-18 18:39:27,987: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 18:39:27,991: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 18:39:27,993: INFO: common: created directory at: artifacts]
[2024-11-18 18:41:25,046: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 18:41:25,049: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 18:41:25,050: INFO: common: created directory at: artifacts]
[2024-11-18 18:45:12,708: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 18:45:12,708: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 18:45:12,713: INFO: common: created directory at: artifacts]
[2024-11-18 19:13:35,258: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 19:13:35,271: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 19:13:35,271: INFO: common: created directory at: artifacts]
[2024-11-18 19:43:19,380: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 19:43:19,384: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 19:43:19,385: INFO: common: created directory at: artifacts]
[2024-11-18 19:53:00,079: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 19:53:00,085: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 19:53:00,087: INFO: common: created directory at: artifacts]
[2024-11-18 21:32:29,889: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-18 21:32:29,889: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-18 21:32:29,889: INFO: common: created directory at: artifacts]
[2024-11-23 17:42:48,280: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-23 17:42:48,294: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-23 17:42:48,296: INFO: common: created directory at: artifacts]
[2024-11-23 17:51:29,383: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-23 17:51:29,386: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-23 17:51:29,388: INFO: common: created directory at: artifacts]
[2024-11-24 16:42:40,370: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-24 16:42:40,380: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 16:42:40,386: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 16:42:40,396: INFO: common: created directory at: artifacts]
[2024-11-24 16:42:40,413: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-24 16:42:40,415: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 16:42:53,624: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 16:42:54,470: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-24 16:54:59,304: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-24 16:54:59,306: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 16:54:59,306: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 16:54:59,308: INFO: common: created directory at: artifacts]
[2024-11-24 16:54:59,308: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-24 16:54:59,309: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 16:55:12,725: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 16:55:13,479: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-24 16:55:13,479: INFO: main: *******************]
[2024-11-24 16:55:13,479: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-24 16:55:13,481: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 16:55:13,482: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 16:55:13,483: INFO: common: created directory at: artifacts]
[2024-11-24 16:55:13,483: INFO: common: created directory at: artifacts/trained_models]
[2024-11-24 16:55:13,484: ERROR: main: "'ConfigBox' object has no attribute 'base_model_path'"]
Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 503, in __getitem__
    return super().__getitem__(item)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'base_model_path'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 536, in __getattr__
    value = self.__getitem__(item, _ignore_default=True)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 524, in __getitem__
    raise BoxKeyError(str(err)) from _exception_cause(err)
box.exceptions.BoxKeyError: "'base_model_path'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 538, in __getattr__
    value = object.__getattribute__(self, item)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ConfigBox' object has no attribute 'base_model_path'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\config_box.py", line 28, in __getattr__
    return super().__getattr__(item)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 552, in __getattr__
    raise BoxKeyError(str(err)) from _exception_cause(err)
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'base_model_path'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 503, in __getitem__
    return super().__getitem__(item)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'base_model_path'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 536, in __getattr__
    value = self.__getitem__(item, _ignore_default=True)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 524, in __getitem__
    raise BoxKeyError(str(err)) from _exception_cause(err)
box.exceptions.BoxKeyError: "'base_model_path'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 538, in __getattr__
    value = object.__getattribute__(self, item)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ConfigBox' object has no attribute 'base_model_path'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 29, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_02_prepare_base_model.py", line 14, in main
    prepare_base_model_config = config.get_prepare_base_model_config()
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\config\configuration.py", line 39, in get_prepare_base_model_config
    base_model_path=Path(config.base_model_path),
                         ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\config_box.py", line 30, in __getattr__
    return super().__getattr__(item.lower())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 552, in __getattr__
    raise BoxKeyError(str(err)) from _exception_cause(err)
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'base_model_path'"
[2024-11-24 17:00:41,356: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-24 17:00:41,364: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 17:00:41,366: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 17:00:41,366: INFO: common: created directory at: artifacts]
[2024-11-24 17:00:41,367: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-24 17:00:41,368: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 17:00:54,477: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 17:00:55,164: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-24 17:00:55,165: INFO: main: *******************]
[2024-11-24 17:00:55,165: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-24 17:00:55,166: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 17:00:55,168: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 17:00:55,168: INFO: common: created directory at: artifacts]
[2024-11-24 17:00:55,169: INFO: common: created directory at: artifacts/trained_models]
[2024-11-24 17:00:55,169: ERROR: main: "'ConfigBox' object has no attribute 'updated_base_model_path'"]
Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 503, in __getitem__
    return super().__getitem__(item)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'updated_base_model_path'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 536, in __getattr__
    value = self.__getitem__(item, _ignore_default=True)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 524, in __getitem__
    raise BoxKeyError(str(err)) from _exception_cause(err)
box.exceptions.BoxKeyError: "'updated_base_model_path'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 538, in __getattr__
    value = object.__getattribute__(self, item)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ConfigBox' object has no attribute 'updated_base_model_path'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\config_box.py", line 28, in __getattr__
    return super().__getattr__(item)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 552, in __getattr__
    raise BoxKeyError(str(err)) from _exception_cause(err)
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'updated_base_model_path'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 503, in __getitem__
    return super().__getitem__(item)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'updated_base_model_path'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 536, in __getattr__
    value = self.__getitem__(item, _ignore_default=True)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 524, in __getitem__
    raise BoxKeyError(str(err)) from _exception_cause(err)
box.exceptions.BoxKeyError: "'updated_base_model_path'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 538, in __getattr__
    value = object.__getattribute__(self, item)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ConfigBox' object has no attribute 'updated_base_model_path'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 29, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_02_prepare_base_model.py", line 14, in main
    prepare_base_model_config = config.get_prepare_base_model_config()
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\config\configuration.py", line 40, in get_prepare_base_model_config
    updated_base_model_path=Path(config.updated_base_model_path),
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\config_box.py", line 30, in __getattr__
    return super().__getattr__(item.lower())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\box\box.py", line 552, in __getattr__
    raise BoxKeyError(str(err)) from _exception_cause(err)
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'updated_base_model_path'"
[2024-11-24 17:02:04,253: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-24 17:02:04,254: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 17:02:04,256: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 17:02:04,256: INFO: common: created directory at: artifacts]
[2024-11-24 17:02:04,257: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-24 17:02:04,257: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 17:02:17,375: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 17:02:18,066: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-24 17:02:18,066: INFO: main: *******************]
[2024-11-24 17:02:18,067: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-24 17:02:18,069: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 17:02:18,071: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 17:02:18,071: INFO: common: created directory at: artifacts]
[2024-11-24 17:02:18,072: INFO: common: created directory at: artifacts/trained_models]
[2024-11-24 17:02:18,072: ERROR: main: PrepareBaseModelConfig.__init__() missing 1 required positional argument: 'updated_base_model_path']
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 29, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_02_prepare_base_model.py", line 14, in main
    prepare_base_model_config = config.get_prepare_base_model_config()
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\config\configuration.py", line 37, in get_prepare_base_model_config
    prepare_base_model_config = PrepareBaseModelConfig(
                                ^^^^^^^^^^^^^^^^^^^^^^^
TypeError: PrepareBaseModelConfig.__init__() missing 1 required positional argument: 'updated_base_model_path'
[2024-11-24 17:03:50,602: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-24 17:03:50,604: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 17:03:50,605: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 17:03:50,605: INFO: common: created directory at: artifacts]
[2024-11-24 17:03:50,606: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-24 17:03:50,606: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 17:04:03,658: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-24 17:04:04,340: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-24 17:04:04,340: INFO: main: *******************]
[2024-11-24 17:04:04,340: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-24 17:04:04,341: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 17:04:04,343: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 17:04:04,344: INFO: common: created directory at: artifacts]
[2024-11-24 17:04:04,344: INFO: common: created directory at: artifacts/trained_models]
[2024-11-24 17:04:04,344: INFO: main: f>>>>> stage {STAGE_NAME} completed <<<<

x===================x]
[2024-11-24 17:04:04,344: INFO: main: **************]
[2024-11-24 17:04:04,345: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-24 17:04:04,346: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-24 17:04:04,347: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-24 17:04:04,348: INFO: common: created directory at: artifacts]
[2024-11-24 17:04:04,348: ERROR: main: EvaluationConfig.__init__() missing 2 required positional arguments: 'params_image_size' and 'params_batch_size']
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 43, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_03_model_evaluation.py", line 17, in main
    evaluation_config = config.get_evaluation_config()
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\config\configuration.py", line 55, in get_evaluation_config
    eval_config = EvaluationConfig(
                  ^^^^^^^^^^^^^^^^^
TypeError: EvaluationConfig.__init__() missing 2 required positional arguments: 'params_image_size' and 'params_batch_size'
[2024-11-25 17:26:47,674: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-25 17:26:47,689: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 17:26:47,711: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 17:26:47,716: INFO: common: created directory at: artifacts]
[2024-11-25 17:26:47,717: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-25 17:26:47,718: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 17:27:01,697: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 17:27:03,713: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-25 17:27:03,714: INFO: main: *******************]
[2024-11-25 17:27:03,715: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-25 17:27:03,720: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 17:27:03,724: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 17:27:03,726: INFO: common: created directory at: artifacts]
[2024-11-25 17:27:03,726: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 17:27:03,727: INFO: main: f>>>>> stage {STAGE_NAME} completed <<<<

x===================x]
[2024-11-25 17:27:03,727: INFO: main: **************]
[2024-11-25 17:27:03,728: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-25 17:27:03,731: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 17:27:03,734: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 17:27:03,736: INFO: common: created directory at: artifacts]
[2024-11-25 17:27:03,847: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 17:27:03,848: ERROR: main: 'PrepareBaseModel' object has no attribute 'load_state_dict']
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 43, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_03_model_evaluation.py", line 25, in main
    model_predictions,images_name=evaluate.make_predictions(dataset_loader, base_model)
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\model_evaluation.py", line 54, in make_predictions
    faster_rcnn_model.load_state_dict(torch.load(self.config.path_of_model, map_location=device))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'PrepareBaseModel' object has no attribute 'load_state_dict'
[2024-11-25 17:39:55,103: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-25 17:39:55,107: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 17:39:55,109: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 17:39:55,110: INFO: common: created directory at: artifacts]
[2024-11-25 17:39:55,110: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-25 17:39:55,112: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 17:40:09,627: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1-6haawpnaPTHxclM7WSrxhiO-Vwx2hRG/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 17:40:11,311: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-25 17:40:11,312: INFO: main: *******************]
[2024-11-25 17:40:11,313: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-25 17:40:11,319: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 17:40:11,322: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 17:40:11,324: INFO: common: created directory at: artifacts]
[2024-11-25 17:40:11,325: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 17:40:11,326: INFO: main: f>>>>> stage {STAGE_NAME} completed <<<<

x===================x]
[2024-11-25 17:40:11,327: INFO: main: **************]
[2024-11-25 17:40:11,327: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-25 17:40:11,336: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 17:40:11,339: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 17:40:11,340: INFO: common: created directory at: artifacts]
[2024-11-25 17:40:11,453: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 17:48:31,556: ERROR: main: No annotations found for image: image_0050.jpg]
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 43, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_03_model_evaluation.py", line 26, in main
    model_predictions, images_name=evaluate.make_predictions(dataset_loader, model)
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\model_evaluation.py", line 60, in make_predictions
    for images, imageids, imagenames, target in dataset_loader:
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\utils\data\dataloader.py", line 633, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\utils\data\dataloader.py", line 677, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\utils\data\_utils\fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\utils\data\_utils\fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
            ~~~~~~~~~~~~^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\electoral_symbol_dataset.py", line 52, in __getitem__
    raise ValueError(f"No annotations found for image: {img_name}")
ValueError: No annotations found for image: image_0050.jpg
[2024-11-25 18:35:24,224: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-25 18:35:24,252: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 18:35:24,267: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 18:35:24,269: INFO: common: created directory at: artifacts]
[2024-11-25 18:35:24,269: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-25 18:35:24,272: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Yak55tdjwSDY8r50HEVGpQSXLpa3wZWy/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 18:35:37,369: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Yak55tdjwSDY8r50HEVGpQSXLpa3wZWy/view?usp=drive_link into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 18:35:37,382: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 14, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\data_ingestion.py", line 42, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file,'r') as zip_ref:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\zipfile.py", line 1312, in __init__
    self._RealGetContents()
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\zipfile.py", line 1379, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2024-11-25 18:38:57,443: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-25 18:38:57,444: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 18:38:57,448: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 18:38:57,448: INFO: common: created directory at: artifacts]
[2024-11-25 18:38:57,448: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-25 18:38:57,450: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 18:39:04,913: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 18:39:05,200: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-25 18:39:05,200: INFO: main: *******************]
[2024-11-25 18:39:05,200: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-25 18:39:05,203: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 18:39:05,203: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 18:39:05,203: INFO: common: created directory at: artifacts]
[2024-11-25 18:39:05,203: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 18:39:05,208: INFO: main: f>>>>> stage {STAGE_NAME} completed <<<<

x===================x]
[2024-11-25 18:39:05,208: INFO: main: **************]
[2024-11-25 18:39:05,208: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-25 18:39:05,212: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 18:39:05,215: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 18:39:05,216: INFO: common: created directory at: artifacts]
[2024-11-25 18:39:05,257: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 18:39:05,897: ERROR: main: [Errno 2] No such file or directory: 'artifacts\\trained_models\\trained_model_fasterrcnn_6_.pth']
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 43, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_03_model_evaluation.py", line 26, in main
    model_predictions, images_name=evaluate.make_predictions(dataset_loader, model)
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\model_evaluation.py", line 54, in make_predictions
    faster_rcnn_model.load_state_dict(torch.load(self.config.path_of_model, map_location=device))
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\serialization.py", line 791, in load
    with _open_file_like(f, 'rb') as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\serialization.py", line 271, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\serialization.py", line 252, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts\\trained_models\\trained_model_fasterrcnn_6_.pth'
[2024-11-25 18:39:21,361: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-25 18:39:21,381: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 18:39:21,385: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 18:39:21,385: INFO: common: created directory at: artifacts]
[2024-11-25 18:39:21,390: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-25 18:39:21,390: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 18:39:28,732: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 18:39:29,038: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-25 18:39:29,038: INFO: main: *******************]
[2024-11-25 18:39:29,038: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-25 18:39:29,041: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 18:39:29,041: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 18:39:29,041: INFO: common: created directory at: artifacts]
[2024-11-25 18:39:29,041: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 18:39:29,045: INFO: main: f>>>>> stage {STAGE_NAME} completed <<<<

x===================x]
[2024-11-25 18:39:29,045: INFO: main: **************]
[2024-11-25 18:39:29,045: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-25 18:39:29,047: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 18:39:29,047: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 18:39:29,051: INFO: common: created directory at: artifacts]
[2024-11-25 18:39:29,096: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 18:39:29,863: ERROR: main: Error(s) in loading state_dict for FasterRCNN:
	size mismatch for roi_heads.box_predictor.cls_score.weight: copying a param with shape torch.Size([45, 1024]) from checkpoint, the shape in current model is torch.Size([44, 1024]).
	size mismatch for roi_heads.box_predictor.cls_score.bias: copying a param with shape torch.Size([45]) from checkpoint, the shape in current model is torch.Size([44]).
	size mismatch for roi_heads.box_predictor.bbox_pred.weight: copying a param with shape torch.Size([180, 1024]) from checkpoint, the shape in current model is torch.Size([176, 1024]).
	size mismatch for roi_heads.box_predictor.bbox_pred.bias: copying a param with shape torch.Size([180]) from checkpoint, the shape in current model is torch.Size([176]).]
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 43, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_03_model_evaluation.py", line 26, in main
    model_predictions, images_name=evaluate.make_predictions(dataset_loader, model)
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\model_evaluation.py", line 54, in make_predictions
    faster_rcnn_model.load_state_dict(torch.load(self.config.path_of_model, map_location=device))
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\nn\modules\module.py", line 2041, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for FasterRCNN:
	size mismatch for roi_heads.box_predictor.cls_score.weight: copying a param with shape torch.Size([45, 1024]) from checkpoint, the shape in current model is torch.Size([44, 1024]).
	size mismatch for roi_heads.box_predictor.cls_score.bias: copying a param with shape torch.Size([45]) from checkpoint, the shape in current model is torch.Size([44]).
	size mismatch for roi_heads.box_predictor.bbox_pred.weight: copying a param with shape torch.Size([180, 1024]) from checkpoint, the shape in current model is torch.Size([176, 1024]).
	size mismatch for roi_heads.box_predictor.bbox_pred.bias: copying a param with shape torch.Size([180]) from checkpoint, the shape in current model is torch.Size([176]).
[2024-11-25 20:10:04,761: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-25 20:10:04,761: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 20:10:04,761: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 20:10:04,761: INFO: common: created directory at: artifacts]
[2024-11-25 20:10:04,761: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-25 20:10:04,761: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 20:10:11,754: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 20:10:12,002: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-25 20:10:12,002: INFO: main: *******************]
[2024-11-25 20:10:12,002: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-25 20:10:12,002: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 20:10:12,002: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 20:10:12,002: INFO: common: created directory at: artifacts]
[2024-11-25 20:10:12,002: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 20:10:12,002: INFO: main: f>>>>> stage {STAGE_NAME} completed <<<<

x===================x]
[2024-11-25 20:10:12,002: INFO: main: **************]
[2024-11-25 20:10:12,010: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-25 20:10:12,011: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 20:10:12,015: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 20:10:12,015: INFO: common: created directory at: artifacts]
[2024-11-25 20:10:12,037: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 20:10:12,949: ERROR: main: No annotations found for image: image_0000.jpg]
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 43, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_03_model_evaluation.py", line 26, in main
    model_predictions, images_name=evaluate.make_predictions(dataset_loader, model)
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\model_evaluation.py", line 60, in make_predictions
    for images, imageids, imagenames, target in dataset_loader:
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\utils\data\dataloader.py", line 633, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\utils\data\dataloader.py", line 677, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\utils\data\_utils\fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\utils\data\_utils\fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
            ~~~~~~~~~~~~^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\electoral_symbol_dataset.py", line 52, in __getitem__
    raise ValueError(f"No annotations found for image: {img_name}")
ValueError: No annotations found for image: image_0000.jpg
[2024-11-25 20:12:03,228: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-25 20:12:03,228: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 20:12:03,244: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 20:12:03,244: INFO: common: created directory at: artifacts]
[2024-11-25 20:12:03,244: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-25 20:12:03,244: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 20:12:10,737: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 20:12:11,030: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-25 20:12:11,032: INFO: main: *******************]
[2024-11-25 20:12:11,032: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-25 20:12:11,032: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 20:12:11,032: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 20:12:11,032: INFO: common: created directory at: artifacts]
[2024-11-25 20:12:11,032: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 20:12:11,032: INFO: main: f>>>>> stage {STAGE_NAME} completed <<<<

x===================x]
[2024-11-25 20:12:11,039: INFO: main: **************]
[2024-11-25 20:12:11,039: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-25 20:12:11,040: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 20:12:11,041: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 20:12:11,042: INFO: common: created directory at: artifacts]
[2024-11-25 20:12:11,061: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 20:44:27,820: ERROR: main: [Errno 2] No such file or directory: 'D:\\Oslo\\OsloMet\\Fourth semester\\Electoral_Symbols_And_Vote_Detection_MLOPS\\Electoral_Symbols_And_Vote_Detection\\output\\vote_validation\\yolo\\no_stamp_image_0037.jpg.jpg']
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 43, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_03_model_evaluation.py", line 32, in main
    evaluate.vote_validation(test_set, images_name, model_predictions)
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\model_evaluation.py", line 90, in vote_validation
    vote_validate.validate_vote(test_set, test_images, predictions, self.true_annotation_labels, self.config.test_images_path)
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\vote_validation\faster_rcnn\validate_vote.py", line 254, in validate_vote
    plt.savefig(f'./output/vote_validation/yolo/no_stamp_{image_name}.jpg', bbox_inches='tight', pad_inches=0, dpi=600)
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\matplotlib\pyplot.py", line 1228, in savefig
    res = fig.savefig(*args, **kwargs)  # type: ignore[func-returns-value]
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\matplotlib\figure.py", line 3395, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\matplotlib\backend_bases.py", line 2204, in print_figure
    result = print_method(
             ^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\matplotlib\backend_bases.py", line 2054, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
                                                                 ^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\matplotlib\backends\backend_agg.py", line 513, in print_jpg
    self._print_pil(filename_or_obj, "jpeg", pil_kwargs, metadata)
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\matplotlib\backends\backend_agg.py", line 445, in _print_pil
    mpl.image.imsave(
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\matplotlib\image.py", line 1676, in imsave
    image.save(fname, **pil_kwargs)
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\PIL\Image.py", line 2600, in save
    fp = builtins.open(filename, "w+b")
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\Oslo\\OsloMet\\Fourth semester\\Electoral_Symbols_And_Vote_Detection_MLOPS\\Electoral_Symbols_And_Vote_Detection\\output\\vote_validation\\yolo\\no_stamp_image_0037.jpg.jpg'
[2024-11-25 21:32:28,589: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-25 21:32:28,607: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 21:32:28,607: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 21:32:28,614: INFO: common: created directory at: artifacts]
[2024-11-25 21:32:28,614: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-25 21:32:28,616: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 21:32:37,427: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-25 21:32:37,971: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-25 21:32:37,971: INFO: main: *******************]
[2024-11-25 21:32:37,971: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-25 21:32:37,985: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 21:32:37,990: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 21:32:37,992: INFO: common: created directory at: artifacts]
[2024-11-25 21:32:37,992: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 21:32:37,992: INFO: main: f>>>>> stage {STAGE_NAME} completed <<<<

x===================x]
[2024-11-25 21:32:37,992: INFO: main: **************]
[2024-11-25 21:32:37,992: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-25 21:32:38,000: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-25 21:32:38,003: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-25 21:32:38,003: INFO: common: created directory at: artifacts]
[2024-11-25 21:32:38,064: INFO: common: created directory at: artifacts/trained_models]
[2024-11-25 22:01:55,636: INFO: main: >>>>>> stage {STAGE_NAME} completed <<<<

x======x]
[2024-11-28 16:52:45,025: ERROR: utils: Form data requires "python-multipart" to be installed. 
You can install "python-multipart" with: 

pip install python-multipart
]
[2024-11-29 21:37:11,814: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-29 21:37:11,814: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-29 21:37:11,830: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-29 21:37:11,830: INFO: common: created directory at: artifacts]
[2024-11-29 21:37:11,830: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-29 21:37:11,830: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-29 21:37:19,159: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-29 21:37:19,661: INFO: main: >>>>>>> Stage Data Ingestion Stage completed <<<<<<<
x==================x]
[2024-11-29 21:37:19,662: INFO: main: *******************]
[2024-11-29 21:37:19,662: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-29 21:37:19,667: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-29 21:37:19,672: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-29 21:37:19,674: INFO: common: created directory at: artifacts]
[2024-11-29 21:37:19,677: INFO: common: created directory at: artifacts/trained_models]
[2024-11-29 21:37:19,679: INFO: main: >>>>> stage Prepare base model stage completed <<<<

x===================x]
[2024-11-29 21:37:19,682: INFO: main: **************]
[2024-11-29 21:37:19,683: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-29 21:37:19,689: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-29 21:37:19,695: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-29 21:37:19,697: INFO: common: created directory at: artifacts]
[2024-11-29 21:37:19,751: INFO: common: created directory at: artifacts/trained_models]
[2024-11-29 21:37:21,191: ERROR: main: [Errno 2] No such file or directory: 'artifacts\\trained_models\\trained_model_fasterrcnn_6_11.pth']
Traceback (most recent call last):
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\main.py", line 43, in <module>
    obj.main()
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\pipeline\stage_03_model_evaluation_with_mlflow.py", line 26, in main
    model_predictions, images_name=evaluate.make_predictions(dataset_loader, model)
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Oslo\OsloMet\Fourth semester\Electoral_Symbols_And_Vote_Detection_MLOPS\Electoral_Symbols_And_Vote_Detection\src\modules\symbol_detection\faster_rcnn\components\model_evaluation.py", line 59, in make_predictions
    faster_rcnn_model.load_state_dict(torch.load(self.config.path_of_model, map_location=device))
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\serialization.py", line 791, in load
    with _open_file_like(f, 'rb') as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\serialization.py", line 271, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\suraj\AppData\Local\Programs\Python\Python311\Lib\site-packages\torch\serialization.py", line 252, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts\\trained_models\\trained_model_fasterrcnn_6_11.pth'
[2024-11-29 21:42:47,227: INFO: main: >>>>>>> Stage Data Ingestion Stage started <<<<<<<]
[2024-11-29 21:42:47,227: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-29 21:42:47,227: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-29 21:42:47,227: INFO: common: created directory at: artifacts]
[2024-11-29 21:42:47,227: INFO: common: created directory at: artifacts/data_ingestion]
[2024-11-29 21:42:47,227: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1N6biugOgu7Id53ayyOMt1j8iKTKYe0Zf/view?usp=sharing into file artifacts/data_ingestion/test_data_set.zip]
[2024-11-29 21:43:19,078: INFO: main: *******************]
[2024-11-29 21:43:19,078: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-29 21:43:19,078: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-29 21:43:19,078: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-29 21:43:19,078: INFO: common: created directory at: artifacts]
[2024-11-29 21:43:19,078: INFO: common: created directory at: artifacts/trained_models]
[2024-11-29 21:43:19,078: INFO: main: >>>>> stage Prepare base model stage completed <<<<

x===================x]
[2024-11-29 21:43:19,078: INFO: main: **************]
[2024-11-29 21:43:19,078: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-29 21:43:19,078: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-29 21:43:19,078: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-29 21:43:19,091: INFO: common: created directory at: artifacts]
[2024-11-29 21:43:19,096: INFO: common: created directory at: artifacts/trained_models]
[2024-11-29 22:14:23,898: INFO: main: >>>>>> stage Model evaluation stage completed <<<<

x======x]
[2024-11-29 22:25:29,292: INFO: main: *******************]
[2024-11-29 22:25:29,292: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-29 22:25:29,301: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-29 22:25:29,302: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-29 22:25:29,303: INFO: common: created directory at: artifacts]
[2024-11-29 22:25:29,303: INFO: common: created directory at: artifacts/trained_models]
[2024-11-29 22:25:29,303: INFO: main: >>>>> stage Prepare base model stage completed <<<<

x===================x]
[2024-11-29 22:25:29,304: INFO: main: **************]
[2024-11-29 22:25:29,304: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-29 22:25:29,305: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-29 22:25:29,306: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-29 22:25:29,307: INFO: common: created directory at: artifacts]
[2024-11-29 22:25:29,322: INFO: common: created directory at: artifacts/trained_models]
[2024-11-29 22:54:08,479: INFO: common: json file saved at: scores.json]
[2024-11-29 22:55:53,522: INFO: main: >>>>>> stage Model evaluation stage completed <<<<

x======x]
[2024-11-30 13:09:53,631: INFO: main: *******************]
[2024-11-30 13:09:53,633: INFO: main: >>>>>> stage Prepare base model stage started <<<<]
[2024-11-30 13:09:53,654: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-30 13:09:53,677: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-30 13:09:53,677: INFO: common: created directory at: artifacts]
[2024-11-30 13:09:53,678: INFO: common: created directory at: artifacts/trained_models]
[2024-11-30 13:09:53,678: INFO: main: >>>>> stage Prepare base model stage completed <<<<

x===================x]
[2024-11-30 13:09:53,678: INFO: main: **************]
[2024-11-30 13:09:53,678: INFO: main: >>>>>>> stage Model evaluation stage started <<<<]
[2024-11-30 13:09:53,683: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-11-30 13:09:53,688: INFO: common: yaml file: params.yaml loaded successfully]
[2024-11-30 13:09:53,688: INFO: common: created directory at: artifacts]
[2024-11-30 13:09:53,742: INFO: common: created directory at: artifacts/trained_models]
[2024-11-30 13:48:37,586: INFO: common: json file saved at: scores.json]
[2024-11-30 13:48:38,081: INFO: main: >>>>>> stage Model evaluation stage completed <<<<

x======x]
